core/src/test/java/org/apache/calcite/test/SqlToRelConverterTest.java

```diff
@@ -1704,6 +1704,23 @@ void checkCorrelatedMapSubQuery(boolean expand) {
     sql(sql).withExpand(false).ok();
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-5156">[CALCITE-5156]
+   * Support implicit type cast for IN Sub-query</a>. */
+  @Test void testInSubQueryWithTypeCast() {
```

Comment:
Maybe the UnitTest can be writtren in TypeCoercionConverterTest
> Just minor comment

Reply:
> Maybe the UnitTest can be written in TypeCoercionConverterTest

+1.

Reply:
Done!